app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,81 @@
+class TasksController < ApplicationController
+
+  def index
+    @tasks = Task.all.order(:id)
+  end
+
+  def show 
+    task_id = params[:id].to_i
+    @task = Task.find_by(id: task_id)
+
+    if @task.nil?
+      redirect_to tasks_path
+    end
+  end
+
+  def new 
+    @task = Task.new
+    @task.task = "I have to... "
+    @task.completed = "Incomplete"
```

Comment:
What data type should `@task.completed` be? Right here you set it to a string (`"Incomplete"`). What logic does the task need to do in order to make task completion functional?